src/main/treeDataProviders/dependenciesTree/dependenciesTreeFactory.ts

```diff
@@ -18,11 +19,41 @@ export class DependenciesTreesFactory {
         quickScan: boolean
     ) {
         if (treesManager.connectionManager.areXrayCredentialsSet()) {
-            await GoUtils.createDependenciesTrees(workspaceFolders, componentsToScan, treesManager, parent, quickScan);
-            await NpmUtils.createDependenciesTrees(workspaceFolders, componentsToScan, treesManager, parent, quickScan);
-            await PypiUtils.createDependenciesTrees(workspaceFolders, componentsToScan, treesManager, parent, quickScan);
-            await MavenUtils.createMavenDependenciesTrees(workspaceFolders, componentsToScan, treesManager, parent, quickScan);
-            await NugetUtils.createDependenciesTrees(workspaceFolders, componentsToScan, treesManager, parent, quickScan);
+            let projectDescriptors: Map<PackageDescriptorType, vscode.Uri[]> = await ScanUtils.locatePackageDescriptors(
+                workspaceFolders,
+                treesManager.logManager
+            );
+            await GoUtils.createDependenciesTrees(
+                projectDescriptors.get(PackageDescriptorType.GO),
+                componentsToScan,
+                treesManager,
+                parent,
+                quickScan
+            );
+            await NpmUtils.createDependenciesTrees(
+                projectDescriptors.get(PackageDescriptorType.NPM),
+                componentsToScan,
+                treesManager,
+                parent,
+                quickScan
+            );
+            if (!!projectDescriptors.get(PackageDescriptorType.PYTHON)) {
+                await PypiUtils.createDependenciesTrees(workspaceFolders, componentsToScan, treesManager, parent, quickScan);
```

Comment:
This line's arguments alignment is horizontal, while this call for the other package manages (see above) is vertical.

Reply:
It is a decision of the Prettier